### PR TITLE
Security audit users need to manage credentials

### PR DESCRIPTION
**Pull request by TomJKing**
Need permissions so can set up MFA for individual IAM users

### CODE REVIEW

File: iam/templates/iam_user_manage_credentials.json.tpl [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            ],
+            "Resource": "*",
+            "Condition": {
+                "BoolIfExists": {
+                    "aws:MultiFactorAuthPresent": "false"
+                }
+            }
+        }
+    ]
+}
```
Comment (vimleshtna) [inline iam/templates/iam_user_manage_credentials.json.tpl:107]: Need to add a new line

File: iam/variables.tf [hunk 2]
```diff
@@ -17,3 +17,6 @@ variable "security_audit_group" {
   description = "group giving 'read only' and 'security audit' permissions"
   default     = "security-audit"
 }
+
+variable "environment" {
+}
```
Comment (vimleshtna) [inline iam/variables.tf:22]: Need to add a new line

**Review by vimleshtna — COMMENTED** [review]
Looks good, just a few minor comments.

File: iam/templates/iam_user_manage_credentials.json.tpl (hunk 1 above)
Comment (TomJKing) [reply iam/templates/iam_user_manage_credentials.json.tpl:107]: showing new line in my code. Think its github being weird

File: iam/variables.tf (hunk 2 above)
Comment (TomJKing) [reply iam/variables.tf:22]: showing new line in my code. Think its github being weird

**Review by vimleshtna — APPROVED** [review]
Looks good, thanks

**Pull request merged**